product_docs/docs/pgd/5/ddl/ddl-role-manipulation.mdx

```diff
@@ -15,6 +15,7 @@ PGD replicates role manipulation statements if [`bdr.role_replication`](/pgd/lat
 enabled (default) and role manipulation statements are run in a PGD-enabled
 database.
```

Comment:
I was a little unclear on how to read this sentence. Does PGD replication role manipulation statements if a) bdr.role_replication is enabled and b) the role manipulation statements are run in a PGD-enabled database? Or is it:

PGD replicates role manipulation statements if [`bdr.role_replication`](/pgd/latest/reference/pgd-settings#bdrrole_replication) is enabled (default). Role manipulation statements are run in a PGD-enabled
database.

Wondering also if this last sentence can be made active. Just overall this is a hard-to-read sentence, and I couldn't quite suss out the meaning, so your guidance with be appreciated.

Reply:
Left over from multiple database support.

If bdr.role_replication is enabled (default), PGD replicates role manipulation statements in PGD-enabled databases.

This page probably needs reviewing by the devs for best practice.